ansible/roles/baselayout/vars/main.yml

```diff
@@ -35,17 +35,17 @@ common_packages: [
 packages: {
   centos6_x64: ['centos-release-scl'], # only available on x86_64, 32-bit is from https://copr.fedorainfracloud.org/coprs/mlampe
   centos6: [
-    'ccache,git2u,gcc-c++,devtoolset-6,sudo', # even need gcc on centos6 so ccache has symlinks
+    'ccache,git2u,gcc-c++,devtoolset-6,sudo,libcap-devel', # even need gcc on centos6 so ccache has symlinks
```

Comment:
We do not build `master`/16.x on centos 6 so we don't need to install the new package there.
https://github.com/nodejs/build/blob/371bb7626abe773a56c7c62e76d0a551ba1b9717/jenkins/scripts/VersionSelectorScript.groovy#L22-L23

Reply:
Thanks, and sorry about being lazy. I'll update this and the others shortly.